### DISCUSSION TITLE
401 unauthorized for Node-exporter servicemonitor

### DISCUSSION THREAD

**Discussion by subhranil05** [prometheus-operator/kube-prometheus#1976] [Q&A]
We are using TLS in our node-exporter in k8.
As authentication we are using tlsConfig in servicemonitors and bearTokenFile as below
still getting 401 unauthorized in dashboard, Please help me to find a solution, thanks
spec:
endpoints:

bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
interval: 15s
port: https
scheme: https
tlsConfig:
insecureSkipVerify: true

**Comment by ghost** [comment]
same issue here

**Reply by subhranil05** [reply]
@tianjiaolaozu check if your node-exporter daemonset is using kube-rbac-proxy container or not.
In my case kube-rback-proxy was old image where api version was conflicting. Upgraded to latest docker image and issue was fixed
